lounge/docker-compose.yml

```diff
@@ -0,0 +1,8 @@
+lounge:
+  image: thelounge/lounge:latest
+  environment:
+    PORT: 5000 # defaults to 9000
+  ports:
+    - "5000:5000"
```

Comment:
What about leaving this out of the file, using the default 9000, and moving this to a documentation piece instead explaining how one can change the port?
Either the [README](https://github.com/thelounge/docker-lounge/blob/master/README.md), [this page](https://thelounge.github.io/docs/deployment/docker.html), or both.

It seems to be the cleanest thing to do to me.